src/components/Editor/index.js

```diff
@@ -405,6 +413,11 @@ class Editor extends PureComponent {
 
     const sourceId = selectedSource ? selectedSource.get("id") : "";
     const line = lineAtHeight(this.state.editor, sourceId, event);
+
+    if (isEmptyLine(line - 1)) {
```

Comment:
Yeah `lineHeight` util function calls `toSourceLine` which adds 1 to the line
see https://github.com/devtools-html/debugger.html/blob/master/src/utils/editor/index.js#L101

It looks like `lineAtHeight` is only used in this file so we can safely
remove the call to`toSourceLine` from https://github.com/devtools-html/debugger.html/blob/master/src/utils/editor/index.js#L126 to fix the issue